docs/deployment/docker-compose.md

```diff
@@ -37,17 +38,18 @@ Run the user similator load test. For more information see [Load Test](#loadtest
 
 <!-- deploy-test-start run-tests -->
 
-    docker run weaveworksdemos/load-test -d 60 -h edge-router -c 3 -r 10
+    docker run --net mynetwork weaveworksdemos/load-test -d 60 -h edge-router -c 3 -r 10
 
 <!-- deploy-test-end -->
 
 ### Cleaning up
 
 <!-- deploy-test-start destroy-infrastructure -->
 
+    docker network rm mynetwork
```

Comment:
This will probably fail, because the other services are still running. A `docker-compose down` will suffice. This will destroy all containers and delete the network (if no other containers are using it).